### PR TITLE
updating the column names to be in sync with spec

### PR DESCRIPTION
**Pull request by mrcnc**
- see the related issue [here](https://github.com/opengeospatial/geopackage/issues/130) for more info

### CODE REVIEW

File: gpkg/gpkg_db.c [hunk 1]
```diff
@@ -154,9 +154,9 @@ static column_info_t gpkg_data_column_constraints_columns[] = {
   {"constraint_type", "TEXT", N, SQL_NOT_NULL | SQL_UNIQUE(1), NULL},
   {"value", "TEXT", N, SQL_UNIQUE(1), NULL},
   {"min", "NUMERIC", N, 0, NULL},
-  {"minIsInclusive", "BOOLEAN", N, 0, NULL},
+  {"min_is_inclusive", "BOOLEAN", N, 0, NULL},
   {"max", "NUMERIC", N, 0, NULL},
-  {"maxIsInclusive", "BOOLEAN", N, 0, NULL},
+  {"max_is_inclusive, "BOOLEAN", N, 0, NULL},
```
Comment (mrcnc) [inline gpkg/gpkg_db.c]: Just realized this doesn't have a closing `"`

**Comment by pepijnve** [comment]
@mrcnc thanks for the PR. Unfortunately I no longer work at Luciad and don't have write access to this repo anymore. I'll reach out to some ex-colleagues to see what the status of this library is.
I've been playing with the idea of forking the repo and reviving the project for a while now. If I don't hear back from anyone I'll probably do that and merge your PR there.
Slightly off-topic but do you now if there are any other things missing/broken in the library wrt the geopackage 1.2 spec?

**Comment by erikdeweerdt** [comment]
The PR is obviously correct with regards to the spec. It is a backwards-incompatible change, however. We may need to support the previous case as well if we don't want preexisting GeoPackage files to suddenly become unreadable.

**Comment by pepijnve** [comment]
@erikdeweerdt the table definitions only affect the database init and verification logic. Changing this will cause a verify error to be reported against geopackage files < 1.1 and won't allow you to create 1.0 compliant files anymore. Probably not that big of a deal.
I'm surprised I still remember all this after 3 years 😄

**Comment by erikdeweerdt** [comment]
@pepijnve yes, true, but we rely on that verification logic to determine if a file is valid and if we should continue to read it. CheckSpatialMetaData() should succeed in order to open the file.

**Comment by pepijnve** [comment]
Got it. In general, the geopackage support code is missing some kind of versioning on the database definitions. Based on your statement you might not actually be able to open 1.1 or 1.2 files at the moment.

**Comment by pepijnve** [comment]
@erikdeweerdt @mrcnc I think PR #2 supersedes this one. It fixes the `gpkg_data_column_constraints` while retaining support for GeoPackage 1.0.

**Comment by mrcnc** [comment]
Thanks @pepijnve and @erikdeweerdt!  It looks like #2 supersedes this PR by preserving backwards compatibility so I'm fine closing it.

**Pull request closed without merging**